src/main/java/org/jenkinsci/plugins/sma/SMARunner.java

```diff
@@ -58,16 +57,17 @@ public SMARunner(EnvVars jobVariables, String prTargetBranch, SMAJenkinsCIOrgSet
         }
         // Configure using pull request logic
         if (!prTargetBranch.isEmpty() && !shaOverride) {
-            deployAll = false;
-            git = new SMAGit(pathToWorkspace, currentCommit, prTargetBranch, SMAGit.Mode.PRB);
-            previousCommit = git.getPrevCommit();
+//            deployAll = false;
```

Comment:
`deployAll` is really only used in cases where the initial commit has been made to the repository, or if the build was requested to be run in the `SMA_DEPLOY_ALL` mode. 

If were not set here to be false, when SMARunner goes to gather all the deployment metadata, it will get all the items from the repository, not just the ones that have changed in PR mode.